docs/diagrams/README.md

```diff
@@ -0,0 +1,30 @@
+# Diagrams
+
+This folder contains diagrams about functionalities of CWL Viewer.
+The source file for the diagrams is stored in a SVG file. We suggest
+contributors to use Inkscape to edit the SVG file.
```

Comment:
@mr-c  :point_up:

Reply:
The source is included as `docs/diagrams/cwlviewer.svg`

Reply:
Did you make this by hand using inkscape? Or is this SVG an export from some other tool?

Reply:
By hand.